tools/pythonpkg/src/include/duckdb_python/arrow_array_stream.hpp

```diff
@@ -21,6 +21,30 @@
 #include <vector>
 
 namespace duckdb {
+
+namespace pyarrow {
+class record_batch_reader : public py::object {
+public:
+	record_batch_reader(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+	static bool check_(const py::handle &object) {
+		return !py::none().is(object);
+	}
+};
+class arrow_table : public py::object {
+public:
+	arrow_table(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+	static bool check_(const py::handle &object) {
+		return !py::none().is(object);
+	}
+};
```

Comment:
The class names and methods don't comply with the standard layout. In particular, class method nomenclature.


For example:
``` cpp

class MyClass {
public:
	MyClass();

	int my_public_variable;

public:
	void MyFunction();

private:
	void MyPrivateFunction();

private:
	void my_private_variable;
};
```

Reply:
Oh cool, happy to fix that, is that documented anywhere?

Reply:
Yes, in the contributions.md at the root folder.

It should also be enforced by the Tidy-Check CI.

Reply:
@pdet I've made the changes that appear to be required per that style, lmk if those are sufficient